ui/admin/app/templates/scopes/scope/edit.hbs

```diff
@@ -3,13 +3,30 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
+{{#if @model.isGlobal}}
```

Comment:
None blocking question: If these breadcrumbs are shown then then ones at line 18 will show as well?

Reply:
using the breadcrumbs::item  component adds an item to the container that is being kept track of by the breadcrumbs service (so nothing is rendered yet). Then the breadcrumbs::container component on line 19 is what renders all the items in the container.